mobile/lib/modules/settings/ui/cleanup_device_settings/manual_device_cleanup.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:immich_mobile/modules/backup/models/backup_state.model.dart';
+import 'package:immich_mobile/modules/backup/providers/backup.provider.dart';
+import 'package:immich_mobile/modules/home/ui/delete_diaglog.dart';
+import 'package:immich_mobile/shared/providers/asset.provider.dart';
+
+class ManualDeviceCleanupWidget extends HookConsumerWidget {
+  const ManualDeviceCleanupWidget({
+    Key? key,
+    required this.backupState,
+  }) : super(key: key);
+
+  final BackUpState backupState;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isUploaded = useState(
```

Comment:
Continuation of the previous comment, `ref.watch` would react when state change without the need of using `useState`